cunumeric/module.py

```diff
@@ -1871,6 +1875,9 @@ def stack(
     if len(shapes) != 1:
         raise ValueError("all input arrays must have the same shape for stack")
 
+    # handle scalar inputs
+    if type(common_info.ndim) is not int:
+        common_info.ndim = 0
```

Comment:
I don't think is necessary. The `check_shape_dtype_without_axis` always returns an int for `ndim` as far as I can tell, and if I comment out this code and run with scalars, this works fine:

```
arrays = (0, 4)
print(cn.stack(arrays))
```

Reply:
I agree. Fixed. Thanks.